index.html

```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
           </ul>
         </div>
       </section>
+      <section class="innerSection" id="portfolioSection">
+        <h2 class="sectionTitle">My Recent Works</h2>
+
+        <!-- my recent works container -->
+        <div class="sectionGrid">
+          <div class="sectionCard">
+            <img
+              class="cardImage"
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
```

Comment:
Kindly for each project in your projects section put the technologies (HTML, CSS, etc... ) inside a `ul` tag each of them inside a `li` tag to make your code look professional ✔😃👍